concepts/05 UI Components/SpeechToText/00 Overview/10 Key Features.md

```diff
@@ -0,0 +1,14 @@
+- **Text Editor Integration**    
+You can integrate the SpeechToText component with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).
+
+- **Language and Localization Support**    
+SpeechToText implements the Web Speech API SpeechRecognition interface, which supports transcription in multiple languages, including English, Spanish, Hindi, and Portuguese. Supported languages may vary depending on the browser.
+
+- **Configurable Transcription**    
+You can configure [continuous transcription](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#continuous) and [interim transcription results](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#interimResults) to customize SpeechToText transcription behaviour. You can also bias SpeechToText results to a specified [array of words and phrases](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#grammars).
+
+- **Text and Icon Customization**    
```

Comment:
We can name this section Appearance Customization
and also mention here different modes: only icon/text and [different styling modes](link)